spec/predicates/human-review-crev.md

```diff
@@ -0,0 +1,156 @@
+# Predicate type: Dependency Reviews (crev)
+
+Type URI: (tentative) https://in-toto.io/attestation/human-review/crev/v0.1
+
+Version: Do we use Crev's versioning? Currently -1? What about their URI?
```

Comment:
Do we have a resolution for this?  If we don't know maybe just stick with v0.1?

Reply:
I'll reach out to the crev developers again.

Reply:
I've added a note for now. The crev developers are open to us reimplementing crev without necessarily being backwards compatible. IMO we should start in a place of compatibility (or close-to-compatibility) and see where we go from there. For now, I've added a note about crev's versioning and updated to using in-toto specific URIs.